clang/test/lab1/akopyan_zal/Test.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/AlwaysInlinePluginAkopyan%pluginext -plugin always-inline %s 2>&1 | FileCheck %s
+
+// CHECK: __attribute__((always_inline)) FuncInline
+int FuncInline(){ return 10; }
+

```

Comment:
Please check that attribute is absent in the following example
```suggestion
void FuncNotInline() {
  {
  if (true) {
    return;
  }
}
  return;
}
```